### PR TITLE
Flatten BodyParser from Future or CompletionStage

### PR DESCRIPTION
**Pull request by cchantep**

### CODE REVIEW

**Comment by cchantep** [comment]
Should be ok now

File: framework/src/play/src/main/java/play/mvc/BodyParser.java [hunk 1]
```diff
@@ -450,4 +453,25 @@ public DelegatingBodyParser(play.api.mvc.BodyParser<B> delegate, Function<B, A>
         }
     }
 
+    /**
+     * A body parser that completes the
```
Comment (gmethvin) [inline framework/src/play/src/main/java/play/mvc/BodyParser.java]: I think you forgot to complete the sentence.

**Comment by gmethvin** [comment]
lgtm

File: framework/src/play/src/main/java/play/mvc/BodyParser.java (hunk 1 above)
Comment (cchantep) [inline framework/src/play/src/main/java/play/mvc/BodyParser.java]: Updated

**Comment by cchantep** [comment]
Would be nice it get merged before the next release

File: framework/src/play/src/main/java/play/mvc/BodyParser.java (hunk 1 above)
Comment (marcospereira) [inline framework/src/play/src/main/java/play/mvc/BodyParser.java]: @cchantep the sentence stills incomplete.

Comment (cchantep) [inline framework/src/play/src/main/java/play/mvc/BodyParser.java]: updated

Comment (gmethvin) [inline framework/src/play/src/main/java/play/mvc/BodyParser.java]: It doesn't look like it's changed in the diff.

**Comment by cchantep** [comment]
Up

**Comment by gmethvin** [comment]
Sorry for the long @cchantep.

**Comment by cchantep** [comment]
Thanks

**Pull request merged**